packages/E2ETest/windows/TreeDumpLibrary/TreeDumpControlViewManager.cs

```diff
@@ -225,10 +246,12 @@ private void UpdateTextBlockText(string text)
         private TextBlock m_textBlock = null;
         private string m_dumpID = "UnknownTest";
         private string m_dumpExpectedText;
-        private bool m_matchDump = false;
+        private bool m_dumpMatchExpected = false;
         private bool m_errStringShowing = false;
         private string m_errString = "";
         private string m_uiaID = null;
 
+        private DispatcherTimer m_timer = null;
```

Comment:
>DispatcherTimer [](start = 16, length = 15)

I'd like you to review the native module implementation. We should not define any variable in ViewManager.
ViewManager is shared by multiple instance of a Control, and the data should be put in the shadow node other than ViewManager.

Reply:
Agree, I took the shortcut here as we only need support one TreeDump control instance per test. I do have work item to use native module once the host interface is refactored, we can revisit by then.

---
In reply to: [359610817](https://github.com/microsoft/react-native-windows/pull/3793#discussion_r359610817) [](ancestors = 359610817)